src/cli/cli_coap.cpp

```diff
@@ -300,6 +300,7 @@ template <> otError Coap::Process<Cmd("set")>(Arg aArgs[])
  * @endcode
  * @par api_copy
  * #otCoapStart
+ * @sa @coap
```

Comment:
Can you do these like this

`@moreinfo{@coap}`

So it appears like this https://openthread.io/reference/cli/commands#netdata_publish_prefix

It's not really a strict CLI/API reference, which is where something after an `@sa` tag goes.

(I think we did some of your previous guide links like `@sa` and it might be worth changing those too, I forgot about the moreinfo tag until just now)

Reply:
@Vyrastas  - Hi Jeff, sure, I may as well update all of them at once, thanks.  I will look through the other references to concepts guides in the other files.

Reply:
@Vyrastas  -  All tags in the CLis that reference Concepts Guide  are now using the @more info tag.  The affected commands are:  coap, tcp, udp, srp client, and srp server.

Reply:
@Vyrastas  -  All tags in the CLis that reference Concepts Guide  are now using the @more info tag.  The affected commands are:  coap, tcp, udp, srp client, and srp server.